lib/vector/diglib/plus_struct.c

```diff
@@ -678,8 +692,69 @@ int dig_Wr_Plus_head(struct gvfile *fp, struct Plus_head *ptr)
         /* can only happen when sizeof(off_t) == 8 */
         ptr->off_t_size = 8;
     }
-    else
-        ptr->off_t_size = 4;
+    else if (ptr->off_t_size == 0) {
```

Comment:
This test is here to skip the second calculation of the topo file size when `dig_Wr_Plus_head()` is called in `dig_write_plus_file()` the second time?

Reply:
Yes, you're right. `dig_Wr_Plus_head()` gets called twice from `dig_write_plus_file()` and the topo size calculation is rather expensive.